numerics/fma_test.cpp

```diff
@@ -0,0 +1,36 @@
+﻿
+#include "numerics/fma.hpp"
+
+#include "glog/logging.h"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "testing_utilities/almost_equals.hpp"
+
+namespace principia {
+namespace numerics {
+
+using base::CPUFeatureFlags;
+using base::HasCPUFeatures;
+using ::testing::Test;
```

Comment:
We generally don't do a `using` of this class, we use the full name when declaring the fixture.  Also, we put the `using` of symbols that start with `::` at the end of the list.